src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/LanguageExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.DotNet.UpgradeAssistant
+{
+    public static class LanguageExtensions
```

Comment:
Does this need to be in the abstractions?

Reply:
Looks like I only used this in the test project. I'll move it there and we can shift it around later if we want access in other locations.